crates/codegen/runtime/cargo/src/runtime/bindings/mod.rs

```diff
@@ -1,10 +1,193 @@
 #[path = "generated/binding_rules.rs"]
 mod binding_rules;
 
-use metaslang_graph_builder::ast;
-pub use metaslang_graph_builder::functions::Functions;
-pub use metaslang_graph_builder::{ExecutionConfig, ExecutionError, NoCancellation, Variables};
+pub mod graph_builder {
```

Comment:
Q: why does this `graph_builder` module need to be public? IIUC, we are wrapping its functionality in the `Bindings` type below. Do we need to expose these at all?

Reply:
We need it to implement the `build-graph` CLI command and we'll need it later to build the graph snapshots. Other than that, it's internal only.

The `build-graph` command is not needed, and in fact, for the bindings' `rules.msgb` file it can longer be used without modifications: the file needs some globals and functions which the CLI command doesn't make available. That is in later commits, it does work fine with this PR's version. In any case the intent of the `build-graph` was to have an intermediate output of the bindings resolution (and allowing users to build arbitrary graphs, but as we discussed, that's out of scope for now).

The graph snapshots (in later commits, after this PR) is something I'd like to keep for debugging purposes, but maybe we can extract the graph through some escape hatch in the bindings API instead.

Reply:
After giving it some thought, I'll remove the `build-graph` command and remove the `graph_builder` export from the bindings module. The `build-graph` command is no longer useful and if we need to, we can create another binary program somewhere else to build an arbitrary graph.

Reply:
I see it was just renamed to `Bindings` in the latest version. Do you still intend to remove it?

Reply:
I did remove it. The `bindings` command does something different: it will output all definitions and references found in the given file.